error.go

```diff
@@ -0,0 +1,66 @@
+package microerror
+
+import (
+	"encoding/json"
+)
+
+// Error is a predefined error structure which purpose is to act as container
```

Comment:
🇬🇧 nit, "whose purpose".

Reply:
Can you explain this? `whose` is for persons AFAIK. `Error` is rather a thing.

Reply:
Whose can be used for inanimate objects. This is the best link I could find.

https://english.stackexchange.com/questions/23541/can-whose-refer-to-an-inanimate-object

Reply:
TIL, see https://github.com/giantswarm/microerror/pull/9.